client/src/test/java/com/paypal/selion/android/sample/AppiumAndroidScrollTest.java

```diff
@@ -50,20 +50,21 @@ public void initElements(){
     @Test
     @MobileTest(appPath = PAGE_OBJECTS_APP_PATH, device = DEVICE_NAME)
     public void testSwipeActions() {
-        UiObject sampleCell = new UiObject("com.paypal.selion.pageobjectsdemoapp:id/TextView13");
+        UiObject sampleCell = new UiObject("android:id/text1");
         UiButton sampleButton = new UiButton("android:id/button1");
         WebDriverWaitUtils.waitUntilElementIsVisible(MENU_LOCATOR);
         menuButton.click(MENU_LOCATOR);
         menuButton.click(MENU_LOCATOR);
+        menuButton.click(MENU_LOCATOR);
         menuButton.click(scrollView);
         scrollView.swipeUp();
         sampleCell.click(sampleButton);
-        Assert.assertEquals("Cell 13", messageBox.getText());
+        Assert.assertNotEquals("You clicked :One", messageBox.getText());
         sampleButton.click();
         scrollView.swipeDown();
-        sampleCell = new UiObject("com.paypal.selion.pageobjectsdemoapp:id/TextView2");
         sampleCell.click(sampleButton);
-        Assert.assertEquals("Cell 2", messageBox.getText());
+        System.out.println(messageBox.getText());
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [System.out.println is used](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2363196529/issues/source?bid=2928425&fileBranchId=3219780#l66)